src/khoj/processor/conversation/gpt4all/chat_model.py

```diff
@@ -161,7 +166,11 @@ def llm_thread(g, messages: List[ChatMessage], model: GPT4All):
     templated_system_message = prompts.system_prompt_llamav2.format(message=system_message.content)
     templated_user_message = prompts.general_conversation_llamav2.format(query=user_message.content)
     prompted_message = templated_system_message + chat_history + templated_user_message
-    response_iterator = model.generate(prompted_message, streaming=True, max_tokens=2000)
-    for response in response_iterator:
-        g.send(response)
+    response_iterator = model.generate(prompted_message, streaming=True, max_tokens=1000, n_batch=256)
```

Comment:
Any specific reason to use different `n_batch` for `converse_offline` (256) vs `extract_question` (128)?

Reply:
No, we can make them the same.